andrea/js/app.js

```diff
@@ -0,0 +1,67 @@
+// buttons //
+var signUp = document.querySelector(".navbar-sign");
```

Comment:
Instead of using `var` get used to use `let` or `const` accordingly https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75#:~:text=%60const%60%20is%20a%20signal%20that,always%20the%20entire%20containing%20function.

Reply:
Also, see comment below